ansible/templates/alltopo.j2

```diff
@@ -0,0 +1,356 @@
+<DeviceMiniGraph xmlns="Microsoft.Search.Autopilot.Evolution" xmlns:i="http://www.w3.org/2001/XMLSchema-instance">
+  <CpgDec>
```

Comment:
can you sepearate cpgdec, dpgdec, MetadataDeclaration into seperate files, so they can be maintained separately?

Reply:
done